### ISSUE TITLE
Document Uploading Frontend

### ISSUE THREAD

**Issue by ibyers** [zachsanchez113/tableau#68]
Currently allows for file to be chosen from filesystem, working on android confirmed. Adding functionality for google drive needed.

**Comment by zachsanchez113** [comment]
Status update: we're sending files from the frontend, but it's being sent as part of the body instead of as form data

**Comment by zachsanchez113** [comment]
Nope, not updating for a file in the body of the request, sending it as a file directly allows me to pull useful attributes from it

**Comment by zachsanchez113** [comment]
Document upload is functional, changing ticket name to better reflect the fact that we're missing the rest
